Examples/Kitura/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:4.2
```

Comment:
is this downgrade necessary?

Reply:
not necessary, but I thought it better communicated that we support 4.2+. I imagined that someone might copy these `Package.swift` files and think they had to use Swift 5.0 based on this